src/components/Header/index.tsx

```diff
@@ -72,29 +72,25 @@ export const Header = ({ auth, children, name }: HeaderProps) => {
           <span className="fa fa-bars" />
         </button>
         <div className="navbar--container">
-          {user && user.name && (
-            <button
-              className="easi-header__username"
-              type="button"
-              onClick={() => {
-                setDisplayDropdown(!displayDropdown);
-              }}
-            >
-              {user.name}
-            </button>
-          )}
           {isAuthenticated ? (
             <div className="easi-header__dropdown-wrapper" ref={dropdownNode}>
               <button
-                aria-label="Expand"
+                aria-label={
```

Comment:
`aria-label` provides a more descriptive action of what happens when you click it.